src/lifecycle.py

```diff
@@ -24,8 +30,75 @@ def stop(stackname, *services):
 @task
 @requires_aws_stack
 @timeit
+@echo_output
 def restart(stackname):
-    lifecycle.restart(stackname)
+    return lifecycle.restart(stackname)
+
+
+@task
+def restart_all(statefile):
```

Comment:
as for location, yes, a separate `tasks` module would help as it's not as generic as the rest of `lifecycle`

Reply:
thanks to `statefile` being passed it's very clear where the local state is and how to reset it

Reply:
could you see this or a generic version of this being used to automatically restart instances weekly?

if they ran in parallel, or batches ran in parallel, it could all be done very quickly.

Reply:
moved to `tasks`